### PR TITLE
Chore/update storybook

### PR DESCRIPTION
**Pull request by gustaveen**
## What?

Update Storybook packages and update setup

https://user-images.githubusercontent.com/6661511/122432482-a75e8b80-cf95-11eb-85f8-0fbd87f8297c.mov

### CODE REVIEW

File: src/client/pages/Landing/Landing.stories.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Landing } from './Landing'
 export default {
   title: 'Landing',
   component: Landing,
+  parameters: { layout: '' },
```
Comment (styrelius) [inline src/client/pages/Landing/Landing.stories.tsx:10]: Just curious - why is this needed? 🤔

**Review by styrelius — APPROVED** [review]
Very nice! 🙌 Just one question really

File: .storybook/main.js [hunk 2]
```diff
@@ -3,11 +3,8 @@ const path = require('path')
 module.exports = {
   stories: ['../src/**/*.stories.tsx'],
   addons: [
-    '@storybook/addon-backgrounds/register',
-    '@storybook/addon-knobs/register',
-    '@storybook/addon-actions/register',
-    '@storybook/addon-viewport/register',
     'storybook-addon-paddings',
+    '@storybook/addon-essentials',
```
Comment (robinandeer) [inline .storybook/main.js:7]: 🙌

**Review by robinandeer — APPROVED** [review]
Very nice indeed! And setup looks cleaner than before :D

File: src/client/pages/Landing/Landing.stories.tsx (hunk 1 above)
Comment (gustaveen) [reply src/client/pages/Landing/Landing.stories.tsx:10]: Good question! 👍 I've added a global setting for layouts in `preview.js`, and the default is `centered` which is nice for most components unless it's more of a page component. So this resets the layout
